packages/react-native-worklets/src/WorkletsModule/NativeWorklets.ts

```diff
@@ -8,14 +8,19 @@ import { WorkletsError } from '../WorkletsError';
 import type { ShareableRef, WorkletRuntime } from '../workletTypes';
 import type { WorkletsModuleProxy } from './workletsModuleProxy';
 
-export interface IWorkletsModule extends WorkletsModuleProxy {}
+export interface IWorkletsModule extends WorkletsModuleProxy {
+  shareableUndefined: ShareableRef<undefined>;
+  shareableNull: ShareableRef<null>;
+}
 
 export function createNativeWorkletsModule(): IWorkletsModule {
   return new NativeWorklets();
 }
 
 class NativeWorklets {
   #workletsModuleProxy: WorkletsModuleProxy;
+  shareableUndefined: ShareableRef<undefined>;
+  shareableNull: ShareableRef<null>;
```

Comment:
Let's keep them as private fields
```suggestion
  #shareableUndefined: ShareableRef<undefined>;
  #shareableNull: ShareableRef<null>;
```